src/sentry/static/sentry/app/views/settings/personal/personalEmails.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const PersonalEmails = React.createClass({
```

Comment:
Let's try to use es6 classes when possible, only use createClass if you need mixins basically.